File: VSWorkSpace/ITCSUnit2Graphics/bumtest.py
from asyncore import read


bumList = [
    "brayden",
    "emily",
    "ryan",
    "sam",
    "rory",
    "kennedy",
    "ray",
    "rowhan"
    ]

def bumtest(name):
    for i in range(len(bumList)):
        if name.lower() == bumList[i]:
            return name + " is a bum"
    return name + " is not a bum"

while 1 == 1 :
    print(bumtest(input("enter a name: ")))

File: VSWorkSpace/ITCSUnit2Graphics/BumTest.java
import java.util.Scanner;
public class BumTest
{
    public static final String[] bumlist = 
    {
        "emily", 
        "brayden", 
        "hayden", 
        "rory", 
        "kennedy", 
        "ray",
        "ryan",
        "rowhan"
    };


    public static String isBum(String string)
    {
        for(int i = 0; i < bumlist.length; i ++)
        {
            if(string.equalsIgnoreCase(bumlist[i]))
            {
                return string + " is a bum!";
            }
        }
        return string + " is not a bum!";
        
    }
    public static void main(String[] args)
    {
        Scanner s = new Scanner(System.in);

        System.out.println("Enter your name:");
        String name = s.nextLine();
        System.out.println(isBum(name));

        s.close();
    }
}